examples/pytorch/mvgrl/README.md

```diff
@@ -0,0 +1,145 @@
+--wd1              float   Weight decay of main model.            Default is 0.0.
+--wd2              float   Weight decay of linear classifier.     Default is 0.0.
+--hid_dim          float   Embedding dimension.                   Default is 512.
+```
+
+## How to run examples
+
+###### Graph Classification
+
+```python
+# Enter the 'graph' directory
+cd graph
+
+# MUTAG:
+python main.py --dataname MUTAG --epochs 20
+
+# PTC_MR:
+python main.py --dataname PTC_MR --epochs 32 --hid_dim 128
+
+# IMDB-BINARY
+python main.py --dataname IMDB-BINARY --epochs 20 --hid_dim 512 --n_layers 2
+
+# IMDB-MULTI
+python main.py --dataname IMDB-MULTI --epochs 20 --hid_dim 512 --n_layers 2
+```
+###### Node Classification
+
+For semi-supervised node classification on 'Cora', 'Citeseer' and 'Pubmed', we provide 2 implementions:
+
+1. full-graph training, see 'main.py', where we contrast the local and global representations of the whole graph.
+2. subgraph training, see 'main_sample.py', where we contrast the local and global representations of a sampled subgraph with fixed number of nodes.
+
+For larger graphs(e.g. Pubmed), it would be hard to calculate the graph diffusion matrix(i.e, PPR matrix), so we try to approximate it with [APPNP](https://arxiv.org/abs/1810.05997), see funtion 'process_dataset_appnp'  in 'node/dataset.py' for details.
+
+```python
+# Enter the 'node' directory
+cd node
+
+# Cora with full graph
+python main.py --dataname cora --gpu 0
+
+# Cora with sampled subgraphs
+python main_sample.py --dataname cora --gpu 0
+
+# Citeseer with full graph
+python main.py --dataname citeseer --wd1 0.001 --wd2 0.01 --epochs 200 --gpu 0
+
+# Citeseer with sampled subgraphs
+python main_sample.py --dataname citeseer --wd2 0.01 --gpu 0
+
+# Pubmed with sampled subgraphs
+python main_sample.py --dataname pubmed --sample_size 4000 --epochs 400 --patience 999 --gpu 0
+```
+
+## 	Performance
+
+We use the same  hyper-parameter settings as stated in the original paper.
+
+##### Graph Classification:
+
+|      Dataset      | MUTAG | PTC-MR |  REDDIT-B  | IMDB-B | IMDB-M |
+| :---------------: | :---: | :----: | :--------: | :----: | :----: |
+| Accuracy Reported | 89.7  |  62.5  |    84.5    |  74.2  |  51.2  |
+|        DGL        | 89.4  |  62.2  | run failed |  73.8  |  51.1  |
+
+* The datasets that the authors used are slightly different from standard TUDatset (see dgl.data.GINDataset) in the nodes' feature dimensions. (e.g. The node features of 'MUTAG' dataset are 11 dimensional vectors instead of 7)
+* We failed to load REDDIT-BINARY dataset with authors' code.
+
+##### Node classification:
+
+|      Dataset      | Cora | Citeseer | Pubmed |
+| :---------------: | :--: | :------: | :----: |
+| Accuracy Reported | 86.8 |   73.3   |  80.1  |
+|    DGL-sample     | 83.2 |   72.6   |  79.8  |
+|     DGL-full      | 83.5 |   73.7   |  OOM   |
+
+* We faied to reproduce the reported accuracy on 'Cora', even with the authors' codes.
+
```

Comment:
I guess you want to mention if the numbers originally reported are based on full graphs or fixed-size subgraphs.